libopenage/renderer/opengl/renderer.cpp

```diff
@@ -26,7 +27,8 @@ GlRenderer::GlRenderer(const std::shared_ptr<GlContext> &ctx,
 	gl_context{ctx},
 	display{std::make_shared<GlRenderTarget>(ctx,
                                              viewport_size[0],
-                                             viewport_size[1])} {
+                                             viewport_size[1])},
+	shared_quad_vao(std::make_shared<GlVertexArray>(ctx)) {
```

Comment:
You should use a curly braced-initializer here and wherever possible. This prevents unintended type conversion.